vizanti_server/public/js/modules/view.js

```diff
@@ -28,8 +28,8 @@ function hasClassInParentChain(element, className) {
 	return hasClassInParentChain(element.parentElement, className);
 }
 
-const MAX_SCALE = 10000;
-const MIN_SCALE = 3.0;
+const MAX_SCALE = 100;
```

Comment:
Yeah, you are correct. We used MAX_SCALE = 100, cause smallest tiles looked pretty good at this level. I've figured out that SCALE is in units of meters/pixel (inverse of what you can see on [this](https://wiki.openstreetmap.org/wiki/Zoom_levels) page in the table). So scale of 100 means 0,01 m/px, so level 19 tile is magnified 30 times I guess, at least on Equator.

But YES, we haven't thought of inspecting TFs. 

We also locked max zoom to level 19 tiles, because level 20 tiles were not loading unfortunately.

Reply:
Yeah OSM stopped supporting level 20 a while back, supposedly it took too much bandwidth to serve it. Some other providers may go down to 20, so it might make sense to try and fetch one of them, and if it returns anything it would use layer 20 as well.